src/components/Layout/SearchList.tsx

```diff
@@ -0,0 +1,105 @@
+import React from "react"
+
+import useFormatMessage from "decentraland-gatsby/dist/hooks/useFormatMessage"
+
+import {
+  AggregatePlaceAttributes,
+  PlaceListOrderBy,
+} from "../../entities/Place/types"
+import watermelonIcon from "../../images/watermelon-icon.svg"
+import locations from "../../modules/locations"
+import { SegmentPlace } from "../../modules/segment"
+import NoResults from "./NoResults"
+import OverviewList from "./OverviewList"
+
+import "./SearchList.css"
+import "./OverviewList.css"
+
+export type SearchListProps = {
+  isLoadingPlaces: boolean
+  isLoadingWorlds: boolean
+  placeResultList?: AggregatePlaceAttributes[]
+  placeTotalResults?: number
+  worldResultList?: AggregatePlaceAttributes[]
+  worldTotalResults?: number
+  search: string
+  handleFavorite: (
+    id: string,
+    place: AggregatePlaceAttributes,
+    tracking: Record<string, string>
+  ) => void
+  handlingFavorite: Set<string>
+}
+
+export default React.memo(function SearchList(props: SearchListProps) {
+  const {
+    placeResultList,
+    placeTotalResults,
+    worldResultList,
+    worldTotalResults,
+    handleFavorite,
+    isLoadingPlaces,
+    isLoadingWorlds,
+    handlingFavorite,
+    search,
+  } = props
+
+  const l = useFormatMessage()
+
+  const hasNoResults =
+    !isLoadingPlaces &&
+    !isLoadingWorlds &&
+    placeTotalResults === 0 &&
+    worldTotalResults === 0
+
+  if (hasNoResults) {
+    return <NoResults search={search} />
+  }
+
+  return (
+    <>
+      <p className="search-results-header">
```

Comment:
can you use [BEM](https://getbem.com/naming/) format: like `search-results__header`